jupyter-covid19/Dockerfile

```diff
@@ -96,5 +96,6 @@ ADD --chown=jovyan:users https://raw.githubusercontent.com/uc-cdis/covid19-tools
 RUN touch /home/$NB_USER/covid19-notebook/peregrine.py
 
 # premade notebooks dependencies
-RUN pip install --upgrade 'pip<20.3' # pip 20.3 causes dependency resolution issues
-RUN pip install -r /home/$NB_USER/covid19-notebook/requirements.txt
+# RUN pip install --upgrade 'pip<20.3' # pip 20.3 causes dependency resolution issues
+# COPY requirements.txt /home/$NB_USER/covid19-notebook/requirements.txt
+# RUN pip install -r /home/$NB_USER/covid19-notebook/requirements.txt --upgrade
```

Comment:
why did you remove this?

Reply:
This is a mistake, let me uncomment this and commit it again. I would still have the line `# RUN pip install --upgrade 'pip<20.3' # pip 20.3 causes dependency resolution issues` commented as it is having issues with new version of python for me